config/checkstyle/checkstyle.xml

```diff
@@ -15,6 +15,12 @@
 
 <module name="Checker">
     &checkerModuleDirectives;
+    <!-- Multiline because sometimes assertions are long and spotless wrap them, so we must handle that situations -->
+    <module name="RegexpMultiline">
+        <property name="format" value="\bassert\b[^:]*;"/>
```

Comment:
```suggestion
        <property name="format" value="\bassert\b[^:;]*;"/>
```

Reply:
Coz regexes are eager by default (extend as far as possible)

Reply:
Or less obviously:
`<property name="format" value="\bassert\b[^:]*?;"/>`

`*?` is a non-eager regex in most engines (`*` extends as little as possible)

Reply:
G R E E D Y

Reply:
Fixed my comment xD you're right, lazy/eager is for evaluation... regexes are lazy/G R E E D Y :ant:

Reply:
Fixed my comment xD you're right, lazy/eager is for evaluation... regexes are lazy/G R E E D Y :ant:

Reply:
Ok, you're right @micpiotrowski, since this is effectively a **forbidden** and not a **required** regex, the current version with just `[^:]` is correct (and my suggestion is not incorrect but rather redundant)

Reply:
Ok, as agreed through another communication channel `\bassert\b[^:]*;` is enough for our purposes, so leave as is